### DISCUSSION TITLE
Reorder Sessions

### DISCUSSION THREAD

**Discussion by bkonia** [isontheline/pro.webssh.net#823] [Ideas]
Can you please support drag/drop to reorder sessions within the sessions list?

**Comment by isontheline** [comment]
I could implement it of course.

Doesn't the alphabetical order doesn't fit your needs?

**Comment by bkonia** [comment]
When you have many sessions open, some may be more frequently used, thus warranting a higher position in the list, making them easier to find.  It's not a huge deal, but I was surprised that I was unable to rearrange the sessions.

**Reply by isontheline** [reply]
Thanks for your feedback 🙏

I also plan to implement favorites : https://github.com/isontheline/pro.webssh.net/issues/611
